datamodel.itop-standard-email-synchro.xml

```diff
@@ -364,6 +364,10 @@ EOF
 						$sAttCode = trim($aMatches[1]);
 						$sValue = trim($aMatches[2]);
 						$aDefaultValues[$sAttCode] = $sValue;
+						if ($sAttCode == 'name')
+						{
+							$aDefaultValues[$sAttCode] .= ' - AUTOCREATED FOR: '.($oEmail->sCallerEmail);
```

Comment:
Like for the attachment PR in `email-reply`, I think this should rather be in a dict entry so it can be localized and moreover customized (the "AUTOCREATED FOR" text, not the values in it)

Reply:
@Molkobain : Thank you. I agree. 

However, before investing the time, I'd like to ask you to check with @jbostoen and your colleagues, if my variant should be the final one, or if you would like to opt for another approach (completely different, or just different by the means of displaying things).

Is that ok?

Reply:
Sure, but that won't be before mid-February :)